content/en/blog/_posts/2022-10-18-kubernetes-1.26-deprecations-and-removals.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: blog
+title: "Kubernetes Removals, Deprecations, and Major Changes in 1.26"
+date: 2022-11-18
+slug: upcoming-changes-in-kubernetes-1-26
+---
+
+**Authors**: Frederico Muñoz
+
+Change is an integral part of the Kubernetes life-cycle: as Kubernetes grows and matures, features may be deprecated, removed, or replaced with improvements for the health of the project. For Kubernetes v1.26 there are several planned: this article identifies and describes some of them, based on the information available at this mid-cycle point in the v1.26 release process, which is still ongoing and can introduce additional changes.
+
+## The Kubernetes API Removal and Deprecation process {#k8s-api-deprecation-process}
```

Comment:
One big deprecation is that k8s 1.26 is not supporting Containerd 1.5 due to removal of CRI v1alpha2 API: https://github.com/kubernetes/kubernetes/pull/110618

Reply:
Thank you, added that to the last version, could you take a look? I have replaced the order so that one now stands out, and the kubelet one is in the list below.